docs/content/slides/pack-your-bags/index.md

```diff
@@ -61,49 +61,359 @@ name: introductions
+  ![you again](/images/pack-your-bags/you-a-developer.jpg)
+  ![it's the journey that matters](/images/pack-your-bags/mcguffin.png)
+]
+
+---
+class: center, middle
+
+# It Runs Happily In The Cloud
+
+---
+# ....your cloud
+.center[
+  ![that's a bingo](/images/pack-your-bags/cloud-bingo.png)
+]
+
+---
+class: center, middle
+
+# Act Two
+
+---
+class: center, middle
+
+# Your Friend Wants To Run It!
+.center[
+  ![your friend, a computer user](/images/pack-your-bags/your-friend-a-user.jpg)
+]
+
+---
+class: center, middle
+
+# That's great! 
+
+# So you write impressive docs
+.center[
+  ![you fight for the users](/images/pack-your-bags/scroll-of-truth.png)
+]
+
+---
+class: center, middle
+
+# Your friend does not thank you...
+
+.center[
+  ![you fight for the users](/images/pack-your-bags/Spongebob-patrick-crying.jpg)
+]
+
+.footnote[http://vignette3.wikia.nocookie.net/spongebob/images/f/f0/Spongebob-patrick-crying.jpg/revision/latest?cb=20140713205315]
+
+---
+class: center, middle
+
+# So you work together...
+
+.center[
+  ![pair programming](/images/pack-your-bags/working-together.jpg)
+]
+
+---
+class: center, middle
+
+# Then you help a few more people...
+
+.center[
+  ![go team](/images/pack-your-bags/go-team.jpg)
+]
+
+---
+class: center, middle
+
+# Suddenly McGuffin has FANS!
+
+.center[
+  ![all the github stars!!!](/images/pack-your-bags/your-fans.jpg)
+]
+
+---
+class: center, middle
+
+# Making everyone do this is not going to cut it...
+
+.center[
+  ![nobody wants to do this](/images/pack-your-bags/scroll-of-truth.png)
+]
 
 ---
-name: sharing
+class: center, middle
+
+# Containers helped us ship our app...
+
+.center[
+  ![ship it](/images/pack-your-bags/container-ship.jpg)
+]
+
+---
+class: center, middle
+
+# But containers don't really solve this...
+
+.center[
+  ![half way there](/images/pack-your-bags/scroll-of-sad-truth.png)
+]
+
+---
+class: center, middle
 
-# Sharing Bundles
+# So what do we do...
 
-OCI (docker) Registries
-Slide 22
+.center[
+  ![this is my thinking face](/images/pack-your-bags/thinking.jpg)
+]
+
+---
+class: middle
+
+# This is the problem CNAB wants to solve
+
+---
+
+# Hashtag Goals
+
+--
+
+* Package All The Logic To Make Your App Happen
+
+--
+
+* Allow Consumer To Verify Everything They Will Install
+
+--
+
+* Distribute Them In Verifiable Way
+
+---
+class: center, middle
+
+# How that works
+
+.center[
+  ![workflow](/images/pack-your-bags/the-workflow.png) ![magic](/images/pack-your-bags/magic.gif)
+]
+
+.footnote[_http://www.reactiongifs.com/magic-3_]
 
 ---
 name: anatomy
+class: center, middle
 
 # Anatomy of a Bundle
 
-Slide 21 from our talk
+.center[
+  ![so what is it](/images/pack-your-bags/anatomy.png)
+]
 
 ---
-# CNAB Specification
+class: center, middle
 
-* bundle format
-* run entrypoint
-* well-defined install/uninstall verbs
-* passing data into a bundle
+# Your App...
 
 ---
-# Breakdown of Azure MySQL Wordpress
+
+# The Invocation Image
+
+--
+
+* MSI for the Cloud?
```

Comment:
Can you tell why this isn't rendering as a normal bullet point? It's showing up super funky with an astrix and in a larger font.